Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -74,7 +74,7 @@ def runSelected(opt):
                      10024.0, #2017 ttbar
                      10224.0, #2017 ttbar PU
                      10824.0, #2018 ttbar
-                     11624.0, #2021 ttbar
+                     11634.0, #2021 ttbar
```

Comment:
can we make the runner throw if the workflow is not available?

Reply:
@slava77 this is a good suggestion, I would manage it independently on this PR

Reply:
given some failures in 2023,2024 in some other tests, should we add at least one to the limited set?